addons/l10n_kz/data/account_account_template_data.xml

```diff
@@ -0,0 +1,2017 @@
+        <field name="note"></field>
+    </record>
+    <record id="a1600" model="account.account.template">
+        <field name="name">Биологические активы</field>
+        <field name="code">1600</field>
+        <field name="reconcile" eval="False"/>
+        <field name="chart_template_id" ref="l10n_kz.chart_template"/>
+        <field name="user_type_id" ref="account.data_account_type_current_assets"/>
+        <field name="note"></field>
+    </record>
+    <record id="a1700" model="account.account.template">
+        <field name="name">Прочие краткосрочные активы</field>
+        <field name="code">1700</field>
+        <field name="reconcile" eval="False"/>
+        <field name="chart_template_id" ref="l10n_kz.chart_template"/>
+        <field name="user_type_id" ref="account.data_account_type_current_assets"/>
+        <field name="note"></field>
+    </record>
+    <record id="a1010" model="account.account.template">
+        <field name="name">Денежные средства в кассе</field>
+        <field name="code">1010</field>
+        <field name="reconcile" eval="False"/>
+        <field name="chart_template_id" ref="l10n_kz.chart_template"/>
+        <field name="user_type_id" ref="account.data_account_type_liquidity"/>
+        <field name="note">учитывается движение денежных средств в кассе в национальной и в иностранных валютах</field>
+    </record>
+    <record id="a1020" model="account.account.template">
+        <field name="name">Денежные средства в пути</field>
+        <field name="code">1020</field>
+        <field name="reconcile" eval="False"/>
+        <field name="chart_template_id" ref="l10n_kz.chart_template"/>
+        <field name="user_type_id" ref="account.data_account_type_liquidity"/>
+        <field name="note">учитывается движение денежных средств в пути и конвертации валюты</field>
+    </record>
+    <record id="a1030" model="account.account.template">
+        <field name="name">Денежные средства на текущих банковских счетах</field>
+        <field name="code">1030</field>
+        <field name="reconcile" eval="False"/>
+        <field name="chart_template_id" ref="l10n_kz.chart_template"/>
+        <field name="user_type_id" ref="account.data_account_type_liquidity"/>
+        <field name="note">учитывается движение денежных средств на текущих банковских счетах в национальной и в иностранных валютах</field>
+    </record>
+    <record id="a1030_0_0" model="account.account.template">
+        <field name="name">Неидентифицированные Платежи</field>
+        <field name="code">1030.0.0</field>
+        <field name="reconcile" eval="False"/>
+        <field name="chart_template_id" ref="l10n_kz.chart_template"/>
+        <field name="user_type_id" ref="account.data_account_type_liquidity"/>
+        <field name="note">прендазначен для использования во временных
+        проводках, которые создаются после загрузки банковской выписки, и
+        существуют до завершения идентификации платежа.</field>
+    </record>
```

Comment:
@william-andre  l10n_multilang basically makes sense if the CoA is legal in English as well or if we want to have English as well because of the international context.  (because in cases of multi-company not every account will speak Russian)  But it is a choice.  If we only put Russian, then it is ok like this.  

I don't think note is used in other localizations and probably not too useful as we don't see it in the interface practically. (better put it in the manifest description)

Reply:
About `note`. It can be used as a reference for developers, but also the field could be added to UI via a studio/module customization. I don't see a problem to keep the data in db even if it's not used by most users. But the ones who finds it useful will be very glad for the possibility to make this magic trick and open the secret door.